spec/lib/disability_compensation/loggers/monitor_spec.rb

```diff
@@ -105,4 +105,36 @@
       )
     end
   end
+
+  describe('#track_526_submission_with_banking_info') do
+    let(:user) { build(:disabilities_compensation_user, icn: '123498767V234859') }
+
+    it 'logs the submission' do
+      expect(monitor).to receive(:submit_event).with(
+        :info,
+        'Form 526 submitted with Veteran-supplied banking info',
+        "#{described_class::SUBMISSION_STATS_KEY}.with_banking_info",
+        user_account_uuid: user.uuid,
+        form_id: described_class::FORM_ID
+      )
+
+      monitor.track_526_submission_with_banking_info(user.uuid)
+    end
+  end
+
+  describe('#track_526_submission_without_banking_info') do
+    let(:user) { build(:disabilities_compensation_user, icn: '123498767V234859') }
+
+    it 'logs the submission' do
+      expect(monitor).to receive(:submit_event).with(
+        :info,
+        'Form 526 submitted without Veteran-supplied banking info',
+        "#{described_class::SUBMISSION_STATS_KEY}.without_banking_info",
+        user_account_uuid: user.uuid,
+        form_id: described_class::FORM_ID
+      )
+
+      monitor.track_526_submission_without_banking_info(user.uuid)
```

Comment:
Any chance you want to add a verification that it increments the stats you expect?
Or nah because it is a thing with the base class, so you can assume it'll "just work"